aws/resource_aws_ecs_task_definition.go

```diff
@@ -108,6 +108,44 @@ func resourceAwsEcsTaskDefinition() *schema.Resource {
 							Optional: true,
 							ForceNew: true,
 						},
+
+						"docker_volume_configuration": {
+							Type:     schema.TypeSet,
```

Comment:
When using `MaxItems: 1` we should prefer `schema.TypeList` instead of `schema.TypeSet` for simplification (e.g. no hash function).